src/app/output/output.component.ts

```diff
@@ -1,15 +1,15 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
+import { Observable, from } from 'rxjs';
+
+import { DataService } from '../data.service';
 
 @Component({
   selector: 'app-output',
   templateUrl: './output.component.html',
   styleUrls: ['./output.component.css']
 })
-export class OutputComponent implements OnInit {
-  outputString: string = "";
-
-  constructor() {}
-
-  ngOnInit(): void {}
+export class OutputComponent {
+  outputString$: Observable<String> = from(this.dataService.output);
```

Comment:
The output string will likely be retrieved from a get call from the data service.  I would suggest until that's implemented, to remove this.